### PR TITLE
Add periodic job for CSI migration on AWS

### PR DESCRIPTION
**Pull request by jsafrane**
Target -broken testgrid tab, hoping to move to informing soon.

cc @openshift/storage

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml [hunk 1]
```diff
@@ -350,6 +350,11 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-csi
+- as: e2e-aws-csi-migration
+  interval: 12h
```
Comment (bertinatto) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:354]: I was wondering if 12 hours isn't too frequent, given that most non-CSI periodic jobs have an interval of 48 hours.

Comment (jsafrane) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:354]: If I understood the system correctly, most of the other jobs are run as part of nightly builds + tests that are run much more often (few times per day, not sure why we call them nightly). This job is not mature yet to get there, but once it is part of nightly, we should indeed reduce the frequency here.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**